### ISSUE TITLE
📌 Issue: Improve “Myth vs. Fact” Page for Clarity, Engagement & Accessibility

### ISSUE THREAD

**Issue by iarmaanx** [Jagrati3/Environment_Animal_Safety_Hub#668]
<img width="1919" height="851" alt="Image" src="https://github.com/user-attachments/assets/3e928e83-7aab-4a6b-b8cb-c919455dc4bc" />

---
---
---

URL: https://jagrati3.github.io/Environment_Animal_Safety_Hub/pages/myth-vs-fact.html

Current Problem:
The page lists myths and facts but can be made more user-friendly, informative, and visually engaging. It currently lacks structure, explanations, sources, and accessibility features that help users better understand and trust the content.

@Jagrati3
